networks/137/aa.json

```diff
@@ -13,5 +13,8 @@
   "paymaster": {
     "type": "off"
   },
-  "session_key_validator": "0x5C06CE2b673fD5E6e56076e40DD46aB67f5a72A5"
+  "session_key": {
+    "kernel_version": "0.2.2",
+    "session_key_validator": "0x5C06CE2b673fD5E6e56076e40DD46aB67f5a72A5"
```

Comment:
maybe just `validator`?

Reply:
`sessionKeyValidator` and `kernelVersion` take different parts in session key registration, so I don't see a reason to store them under the name `validator`